### ISSUE TITLE
accuracy

### ISSUE THREAD

**Issue by 1525898173** [LMD0311/PointMamba#47]
Using your pre-trained model, the reproduced results of ScanNetNN do not reach the accuracy reported in the paper, and the discrepancy is quite significant. Could you please explain what might be causing this?

**Comment by LMD0311** [comment]
Thanks for the feedback. You can re-download the re-uploaded pre-trained weight [ckpt](https://github.com/LMD0311/PointMamba/releases/download/ckpts-2/pretrain.pth) and I have conducted a quick reproduction on ScanObjectNN `Object background` and `Object Only` and got close (even higher) results with the following log and weights, which you can test directly:

|                     | Acc.  |      |
| ------------------- | ----- | ---- |
| `Object background` | 94.84 | [ckpt](https://github.com/LMD0311/PointMamba/releases/download/ckpts-2/reprodece_scan_objbg.pth)/[log](https://github.com/user-attachments/files/19339873/20250319_064439.log)  |
| `Object Only`       | 92.77 |  [ckpt](https://github.com/LMD0311/PointMamba/releases/download/ckpts-2/reprodece_scan_objonly.pth)/[log](https://github.com/user-attachments/files/19339891/20250319_105808.log) |


Additionally, I find that different machines or development environments may cause fluctuations in results, in my experiments I used a single NVIDIA 4090 GPU.
